src/irr/asset/CPLYMeshWriter.cpp

```diff
@@ -51,6 +51,16 @@ static asset::E_FORMAT getCorrespondingIntegerFormat(asset::E_FORMAT _fmt)
 }
 }
 
+void flipNormalsAndPositions(asset::ICPUMeshBuffer* _mbuf, uint32_t ix)
+{
+    core::vectorSIMDf tmpValue;
+
+    _mbuf->getAttribute(tmpValue, 0, ix);
+    tmpValue[0] = -tmpValue[0];
+    _mbuf->getAttribute(tmpValue, 3, ix);
+    tmpValue[0] = -tmpValue[0];
```

Comment:
?

Reply:
can you please test before bothering to open a PR?

Reply:
like what is this even doing?

Reply:
it isn't setting the attribute value, to fix